app/router.js

```diff
@@ -148,6 +149,7 @@ router.map(function() {
   this.route('orders', function() {
     this.route('new');
   });
+  this.route('pages', { path: 'pages/:page_id/:page_title' });
```

Comment:
The page ID shouldn't be needed. The implementation should be such that the path the users entered is all that is needed to view the page.

Reply:
@niranjan94 The URL entered by user should be our dyanamic path. Is that what is required here? Is it intended that the user supplies something like "/terms", "/privacy" in the URL input field present at the admin/content/pages?

Reply:
@niranjan94 We dont want to show the page id in the URL but we have to pass it since we have to query the particular page and show its info out there when the user clicks any footer page.  We have to do something like:
```
          {{#link-to 'pages' page.id page.title class='item'}}
            {{page.name}}
          {{/link-to}}

```
and
```
  this.route('pages', { path: 'pages/:page_id/:page_title' })
```
Also, if we dont pass the id, how would we query the details then?